book-1-orientation/chapters/QUEST.md

```diff
@@ -0,0 +1,287 @@
+                    "What's the answer to life, the universe and everything?", 42, 25);
+                Challenge whatSecond = new Challenge(
+                    "What is the current second?", DateTime.Now.Second, 50);
+
+                int randomNumber = new Random().Next() % 10;
+                Challenge guessRandom = new Challenge("What number am I thinking of?", randomNumber, 25);
+
+                Challenge favoriteBeatle = new Challenge(
+                    @"Who's your favorite Beatle?
+        1) John
+        2) Paul
+        3) George
+        4) Ringo
+    ",
+                    4, 20
+                );
+
+                // "Awesomness" is like our Adventurer's current "score"
+                // A higher Awesomeness is better
+
+                // Here we set some reasonable min and max values.
+                //  If a Adventurer has an Awesomeness greater than the max, they are truly awesome
+                //  If a Adventurer has an Awesomeness less than the min, they are terrible
+                int minAwesomeness = 0;
+                int maxAwesomeness = 100;
+
+                // Make a new "Adventurer" object using the "Adventurer" class
+                Adventurer theAdventurer = new Adventurer("Jack");
+
+                // A list of challenges for the Adventurer to complete
+                List<Challenge> challenges = new List<Challenge>()
+                {
+                    twoPlustwo,
+                    theAnswer,
+                    whatSecond,
+                    guessRandom,
+                    favoriteBeatle
+                };
+
+                // Loop through all the challenges and subject the Adventurer to them
+                foreach (Challenge challenge in challenges)
+                {
+                    challenge.RunChallenge(theAdventurer);
+                }
+
+                // This code examines how Awesome the Adventurer is after completing the challenges
+                // And praises or humiliates them accordinly
+                if (theAdventurer.Awesomness >= maxAwesomeness)
+                {
+                    Console.WriteLine("YOU DID IT! You are truly awesome!");
+                }
+                else if (theAdventurer.Awesomness <= minAwesomeness)
+                {
+                    Console.WriteLine("Get out of my sight. Your lack of awesomeness offends me!");
+                }
+                else
+                {
+                    Console.WriteLine("I guess you did...ok? ...sorta. Still, you should get out of my sight.");
+                }
+            }
+        }
+    }
+    ```
+
+    > `Adventurer.cs`
+
+    ```cs
+    namespace Quest
+    {
+        // An instance of the Adventurer class is an object that will undergo some challenges
+        public class Adventurer
+        {
+            // The is an "immutable" property. It only has a "get".
+            // The only place the Name can be set is in the Adventurer constructor
```

Comment:
Personally, I would add 'defined on line 121' to the end of this comment. Yes line 120 has a comment stating that 121 is the constructor, but they will read line 113 first. Anyone lost on the vocab here is only going to feel more frustrated rather than enlightened.